src/cromshell/status/command.py

```diff
@@ -0,0 +1,138 @@
+import fileinput
+import json
+import logging
+import re
+
+import click
+import jq
+import requests
+
+from cromshell.utilities import CromshellConfig, HTTPUtils, IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.argument("workflow_id")
+@click.pass_obj
+def main(cromshell_config, workflow_id):
+    """Check the status of a Cromwell job UUID"""
+    click.echo("status")
+    LOGGER.info("status")
+
+    ret_val = 0
+    request_out = "str"
+
+    # Set cromwell server using submission file. Running the function below with passing only the workflow id
```

Comment:
not a comment do be addressed in this PR, but I'm thinking we should have argument validation checks in future PRs.

Reply:
Added to [project SOW](https://docs.google.com/document/d/1QBZIY4kVMVq_V0DldndHTauBGt3QoC2hjQnX292UlLI/edit#) for now